CODEOWNERS

```diff
@@ -0,0 +1 @@
+/   @hashicorp/github-nomad-core @hashicorp/nomad-eng
```

Comment:
Sorry, late to this PR, but I'm pretty sure this needs to be `*`. 

Example:
https://github.com/hashicorp/nomad-autoscaler/blob/main/.github/CODEOWNERS#L1-L2

Reply:
Oops! Will fix.